.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
```

Comment:
Please don't include a .gitignore; these platform-specific ignores should be in global `~/.gitignore`, not in per-project settings.